Yoast/wordpress-seo
# Used yoast wp data

## Summary

Some minor stuff missing from #9500. the way wp.data was called was wrong, unit-tests failed and some imports were unnecessary. 

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* Used yoast._wp instead of wp because we can't be sure WP is there (instructed by Anton).
* Removed dependencies which turned out to be unnecessary. discussed with Igor.

## Test instructions

This PR can be tested by following these steps:

* Check if the unit-tests pass.
* Perhaps check if issue #9500 is still fixed as originally created.

## Quality assurance


Fixes #9571
